### PR TITLE
fix(bitso) - safeNetwork removal

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

File: ts/src/bitso.ts [hunk 1]
```diff
@@ -1722,7 +1714,7 @@ export default class bitso extends Exchange {
             'txid': this.safeString (details, 'tx_hash'),
             'timestamp': this.parse8601 (datetime),
             'datetime': datetime,
-            'network': this.safeNetwork (networkId),
+            'network': this.networkIdToCode (networkId),
```
Comment (carlosmiei) [inline ts/src/bitso.ts:1717]: same thing here, this is a different behavior

Comment (ttodua) [reply ts/src/bitso.ts]: pushed

File: ts/src/bitso.ts [hunk 2]
```diff
@@ -1722,7 +1714,7 @@ export default class bitso extends Exchange {
             'txid': this.safeString (details, 'tx_hash'),
             'timestamp': this.parse8601 (datetime),
             'datetime': datetime,
-            'network': this.safeNetwork (networkId),
+            'network': this.networkIdToCode (networkId).toUpperCase (),
```
Comment (carlosmiei) [inline ts/src/bitso.ts:1717]: same thing here, what if this is undefined?

Comment (ttodua) [reply ts/src/bitso.ts]: anyway fixed & pushed, but still lmk your answer on above

**Comment by ttodua** [comment]
@carlosmiei this too ^

**Comment by ttodua** [comment]
@carlosmiei and this

**Pull request merged**